package.json

```diff
@@ -26,7 +27,7 @@
     "test": "hardhat test",
     "test:system": "NODE_ENV=system-test hardhat test ./test/system/*.test.js",
     "postinstall": "./scripts/prepare-dependencies.sh $INIT_CWD",
-    "prepack": "tsc -p tsconfig.export.json && hardhat export-artifacts export/artifacts",
+    "prepack": "tsc --target es5 -p tsconfig.export.json && hardhat export-artifacts export/artifacts",
```

Comment:
Can we set the target in `tsconfig.export.json` instead of adding `--target es5` arg here?

Reply:
Sure, I'll change it now.